src/modules/gateway/sagas.ts

```diff
@@ -39,16 +38,16 @@ import {
 } from './actions'
 import { MoonPay } from './moonpay'
 import { MoonPayTransaction, MoonPayTransactionStatus } from './moonpay/types'
-import { getPendingPurchase } from './selectors'
+import { getPendingManaPurchase } from './selectors'
 import { Transak } from './transak'
-import { ManaFiatGatewaySagasConfig } from './types'
+import { ManaFiatGatewaySagasConfig, Purchase, PurchaseStatus } from './types'
 import { purchaseEventsChannel } from './utils'
 
 const DEFAULT_POLLING_DELAY = 3000
 const BUY_MANA_WITH_FIAT_FEEDBACK_MODAL_NAME = 'BuyManaWithFiatFeedbackModal'
 
-export function createManaFiatGatewaysSaga(config: ManaFiatGatewaySagasConfig) {
-  return function* manaFiatGatewaysSaga(): IterableIterator<ForkEffect> {
+export function createGatewaySaga(config: ManaFiatGatewaySagasConfig) {
```

Comment:
Rename to `createGatewaysSaga` (Was in plural before)

Reply:
It was a typo. It should be `create{Name of the module}Saga`. In this case the module is `gateway`.

Reply:
But then it creates the `gatewaysSaga` 🤔